<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Custom</class>
 <widget class="QDialog" name="Custom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="testdefinition_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Test Definition</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_definition">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type: Test</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>1131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="normal_radio_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="parallel_radio_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Parallel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Execution Mode:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="local_radio_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Local</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="eth_radio_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ethernet P2P</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Device mode:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="test_id_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto generated</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_id_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="log_folder_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Logs folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="log_folder_edit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>output/logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_path">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dir</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel_test_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="run_test_button">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>650</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUN</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>1131</width>
     <height>421</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1129</width>
      <height>419</height>
     </rect>
    </property>
    <widget class="QTableView" name="definition_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1131</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>run_test_button</sender>
   <signal>clicked()</signal>
   <receiver>Custom</receiver>
   <slot>runDef()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_test_button</sender>
   <signal>clicked()</signal>
   <receiver>Custom</receiver>
   <slot>cancelRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eth_radio_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>Custom</receiver>
   <slot>selectDeviceRunEth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1050</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>local_radio_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>Custom</receiver>
   <slot>selectDeviceRunLocal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>940</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cancelRun()</slot>
  <slot>runDef()</slot>
  <slot>selectDeviceRunEth()</slot>
  <slot>selectDeviceRunLocal()</slot>
 </slots>
</ui>
